release

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env php
+<?php
+
+/**
+ * This script aims to simplify the release process of AsyncAws components
+ * Unless you are a maintainer of AsyncAws, their is nothing interesting in there.
+ *
+ * This script will search for components to release (based on change log and will)
+ * - Find the next version
+ * - Update "branch-alias" section of the composer.json (for minor updates)
```

Comment:
Updating the branch alias at release time is wrong and too late. The minor version should be bumped only when new features are actually merged, and right at the same time, and the similarly for the major version, at the point where breaking changes are merged, otherwise there is little point in the branch aliases, and we may be better to remove them if we are not wanting to have them accurately reflect the in progress version?